parts/k8s/kubernetes_mountetcd.sh

```diff
@@ -27,4 +27,5 @@ then
     /sbin/sgdisk --new 1 $DISK
     /sbin/mkfs.ext4 $PARTITION -L etcd_disk -F -E lazy_itable_init=1,lazy_journal_init=1
 fi
-mount $MOUNTPOINT
\ No newline at end of file
+mount $MOUNTPOINT
+/bin/chown -R etcd:etcd /var/lib/etcddisk
```

Comment:
Moved this out of cloud-init because it makes sense organizationally to be here!